client/blocks/term-tree-selector/index.jsx

```diff
@@ -6,17 +6,14 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
-import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import TermTreeSelectorTerms from './terms';
 import TermSelectorAddTerm from './add-term';
 
-export default class extends React.Component {
-	static displayName = 'TermTreeSelector';
-
+export default class TermTreeSelector extends React.Component {
```

Comment:
huh, why is this in blocks? looks more like something for `components`. No redux binding.

not for this PR though. :)

Reply:
Ah, the kids (`TermSelectorAddTerm`, `TermTreeSelectorTerms`) are bound.